src/ticketmap/index.tsx

```diff
@@ -76,7 +80,8 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
       tooltipZoneId: '',
       tooltipX: 0,
       tooltipY: 0,
-      ticketGroups: this.props.ticketGroups
+      ticketGroups: this.props.ticketGroups,
+      displayNotFoundImage: false
```

Comment:
I'd like to keep our state properties from becoming view modifiers. `displayNotFoundImage` reads as a sort of manual control of a view element. I would prefer if we renamed it to some description of the app's _state_, such as `mapFetchFailed`. `render()` then has the ability to generate whatever view seems necessary (in this case, creating a not found image).

```suggestion
      mapFetchFailed: false
```

Reply:
I changed it to `mapNotFound` to correlate with the error